openedx/features/course_experience/views/course_outline.py

```diff
@@ -120,3 +130,13 @@ def get_content_milestones_old(self, request, course_key):
         }
 
         return course_content_milestones
+
+    def user_enrolled_after_completion_collection(self, user, course_key):
+        begin_collection_date = datetime.datetime(2018, 01, 24, tzinfo=pytz.utc)
```

Comment:
Could you please some comment behind this constant date and explain that this filter should also be temporary?